### PR TITLE
lxqt-config-monitor: Unintrusive (hopefully) UI improvements proposal

### PR DESCRIPTION
**Pull request by vc-01**
Greenlit by project team member.

Please see individual commit messages for detailed explanations for each change.

Refs: gh-577

### CODE REVIEW

**Comment by tsujan** [comment]
Thanks!

Since you were worried about your PR, I'm here to emphasize that it'll be reviewed (if no other dev volunteers, I'll do it). But since LXQt 1.4.0 will be released in a week or so, and we won't have enough time for testing until then, reviewing/merging will be done after the new release.

**Comment by vc-01** [comment]
Thank you for your very fast response. I'm more than fine if the review takes, say a full year of time for example, after all this is an unpaid project and I'm happy that it exists at all. I was worried more about going against ideas of the project and waste time working on something to be later rejected or frowned upon. Which is especially a case for no-one-need-it-no-one-want-it UI changes. And as mentioned already, the UI for lxqt-config-monitor needs much more than what's been proposed in this PR.

_-- No rush. --_

File: lxqt-config-monitor/managesavedsettings.cpp [hunk 1]
```diff
@@ -164,6 +167,14 @@ void ManageSavedSettings::onApplyItem()
     if (ui.allConfigs->currentItem() == nullptr)
         return;
     MonitorSavedSettings settings = ui.allConfigs->currentItem()->data(Qt::UserRole).value<MonitorSavedSettings>();
+
+    if (not isHardwareCompatible(settings)) {
```
Comment (tsujan) [inline lxqt-config-monitor/managesavedsettings.cpp:171]: Please change this to `if (!isHardwareCompatible(settings)) {`. In LXQt we don't use `and`, `or` or `not`.

Comment (vc-01) [reply lxqt-config-monitor/managesavedsettings.cpp:171]: Sure.

**Comment by tsujan** [comment]
Thanks! IMO, this is much better than the current master branch. There are other problems in the config dialog too, but this PR does its job well. So, if other members find no issue in it either, it could be merged.

Before I give it a "GMT", would you please squash all commits?

**Comment by vc-01** [comment]
> ... would you please squash all commits?

I have squashed the commits as requested.

**Comment by tsujan** [comment]
Assuming that there's no objection, I'm merging it — its goals are reasonable, and it works as intended.

**Pull request merged**